docs/src/main/paradox/running-the-application.md

```diff
@@ -0,0 +1,30 @@
+Running the application
+-----------------------
+
+When you ran the example for the first time, you were able to create and retrieve multiple users. Now that you understand how the example is implemented, let's confirm that the rest of the functionality works. We want to verify that:
+
+* If we try to retrieve users when none exist, we get an empty list.
+* If we try to retrieve a specific user that doesn't exist, we get an informative message.
+* We can delete users.
+
+To test this functionality, follow these steps. If you need reminders on starting the app or sending requests, refer to the @ref:[instructions](index.md#exercising-the-example) in the beginning.
+
+1. If the Akka HTTP server is still running, stop and restart it.
+2. With no users registered, use your tool of choice to:
+3. Retrieve a list of users. Hint: use the `GET` method and append `/users` to the URL.
+
+You should get back an empty list: `{"users":[]}`
+
+4. Try to retrieve a single user named `MrX`. Hint: use the `GET` method and append `users/MrX` to the URL.
```

Comment:
Each of these numbered items with paragraphs in-between becomes a separate list starting at "1" when rendered.